agents/host-arm64/agent-config.yaml

```diff
@@ -0,0 +1,39 @@
+prometheus:
+  global:
+    scrape_interval: 30s
+  wal_directory: /tmp/wal/
+  configs:
+    - name: default
+      scrape_configs:
+        - job_name: file-sd-targets
+          file_sd_configs:
+          - files:
+            - '/etc/opsverse/targets/*.json'
+          relabel_configs:
+            - source_labels: [__address__]
+              regex: '.*'
+              target_label: instance
+              replacement: __HOST__
+      remote_write:
+        - url: https://__METRICS_HOST__/api/v1/write
+          basic_auth:
+            username: devopsnow
+            password: __PASSWORD__
+
+loki:
```

Comment:
This looks to be the config that we used to use with the old version of the agent. This might work with the new one, but for sake of consistency, we may want to use the configs used by `docker-compose` (Note: it used `logs` instead of `loki`)